src/avram/charms/json_extensions.cr

```diff
@@ -6,8 +6,13 @@ module JSON::Serializable
   end
 
   module Lucky(T)
+    alias ColumnType = JSON::Any
```

Comment:
What made this necessary?

Reply:
Oh, that might not be... I was having issues with the Criteria, so I just started adding a ton of stuff until it worked. I'll see if I can remove it.